lib/chef-dk/command/generator_commands/cookbook.rb

```diff
@@ -77,6 +85,7 @@ def initialize(params)
         def run
           read_and_validate_params
           if params_valid?
```

Comment:
This is the intended behavior. We only want to print the usage block when an invalid option is passed or no cookbook name is passed.